resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -207,23 +208,38 @@ cloudwatch.logs.log_group_title=Group: {0}
 cloudwatch.logs.log_stream_does_not_exist=Log stream {0} does not exist
 cloudwatch.logs.log_stream_title=Stream: {0}
 cloudwatch.logs.log_streams=Log Streams
+cloudwatch.logs.no_end_date=End Date must be specified
 cloudwatch.logs.no_events=No Events in Log Stream
 cloudwatch.logs.no_events_query=No Events matching the query found in Log Stream {0}
 cloudwatch.logs.no_log_streams=No Log Stream found
+cloudwatch.logs.no_query_entered=Query must be specified
+cloudwatch.logs.no_query_selected=Query must be entered
```

Comment:
What is wrong with using the message above?

Reply:
Changed the components of the messages. The 2nd one should inform the user to either enter a search term or a query.